index.html

```diff
@@ -4,32 +4,27 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title> My Portfolio</title>
+  <title>Portfolio</title>
   <link rel="stylesheet" href="style.css">
   <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Playfair+Display" />
   <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Poppins" />
 </head>
 <body>
   <div id="nav-wrapper">
     <nav>
-
-      
       <ul>
         <li><a href="#" rel="access">Hello</a></li>
         <li><a href="#" rel="acess">Portfolio</a></li>
         <li><a href="#" rel="access">About</a></li>
         <li><a href="#" rel="access">Contact</a></li>
```

Comment:
- [ ] Optional - I highly recommend that adding corresponding sources to the navigation links so that it can take user the intended section of the page.

Reply:
in the future i will